src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/pci-dss.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: PCI DSS
+tags:
+  - Security
+  - Security and Privacy
+  - Compliance
+metaDescription: Criteria and compliance with PCI DSS audits by New Relic services.
+freshnessValidatedDate: 2024-07-25
+redirects:
+  - /docs/security/security-privacy/compliance/certificates-standards-regulations/PCI DSS-moderate
+---
+
+The Payment Card Industry Data Security Standard (“PCI DSS”), maintained by the Security Standards Council, is a set of security requirements to protect cardholder data environments (“CDEs”) where payment card data is stored, processed, or transmitted. PCI DSS provides a baseline of rigorous technical and operational requirements designed to protect CDEs.
+
+Assessed against the PCI DSS version 4.0, New Relic maintains a Report on Compliance (“ROC”) and Attestation of Compliance (“AOC”) as a [Level 1 Service Provider](https://www.pcisecuritystandards.org/glossary/service-provider/).
+
+New Relic removes some of your sensitive data in logs with automatic [log obfuscation](https://docs.newrelic.com/docs/logs/get-started/new-relics-log-management-security-privacy/#auto-obfuscation), which is enabled by default for all customers and helps you protect your data.
```

Comment:
....log obfuscation. Enabled by default. 

Remove "and helps you protect....", its redundant.